includes/class-reader-activation.php

```diff
@@ -108,6 +110,75 @@ public static function enqueue_scripts() {
 		);
 	}
 
+	/**
+	 * Get settings config with default values.
+	 *
+	 * @return mixed[] Settings default values keyed by their name.
+	 */
+	private static function get_settings_config() {
+		$settings_config = [
+			'enabled'                     => true,
+			'enabled_account_link'        => false,
```

Comment:
IMO if the general Reader Activation features are enables by default, the account link should also be. I imagine a publisher turning on Reader Activation will want the full package right away, without having to click twice.

Reply:
We could have it enabled as part of a default in the onboarding wizard, but not as the default value. Once we launch the functionality (remove the experimental flag) it will force every site to adjust its header UI to accommodate a compulsory new link.